pkgs/applications/graphics/sane/config.nix

```
@@ -33,6 +33,14 @@ stdenv.mkDerivation {
   phases = "installPhase";
 
   installPhase = ''
+    function symlink () {
+      local target=$1 linkname=$2
+      if [ -e "$linkname" ]; then
+        echo "warning: conflict for $linkname. Overriding."
```

Maybe change this warning so it's clear what is being overridden?

```suggestion
        echo "warning: conflict for $linkname. Overriding with $target"
```

---

I wanted to do that, then changed my mind because it is very confusing to only have $target. Maybe `Overriding $(readlink) with $target` ?

---

Sure, that sounds good!